recipes/drf-haystack/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "drf-haystack" %}
+{% set version = "1.8.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: afb0c1384458c4c59afa6946e58e446ae44152081fb3000b8f43d4a13073fb2b
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - django
+    - django-haystack
+    - djangorestframework
```

Comment:
Can you add version constraints from https://github.com/inonit/drf-haystack/blob/master/setup.py#L36-L39

Reply:
Also need python-dateutil

Reply:
Specially the `Django` constraint or this package will be broken with `Django 2.0`.

Reply:
To make it easier here they are:

```
        "Django>=1.11,<2.1",
        "djangorestframework>=3.7",
        "django-haystack>=2.8",
        "python-dateutil"
```

Reply:
I think it allows django 2.0, just not future versions of django